src/components/material/MaterialReviews.tsx

```diff
@@ -1,30 +1,63 @@
 import React from "react";
-import {
-  ExternalReview,
-  InfomediaReview,
-  LibrariansReview
-} from "../../core/dbc-gateway/generated/graphql";
+import { useGetReviewManifestationsQuery } from "../../core/dbc-gateway/generated/graphql";
+import { Pid } from "../../core/utils/types/ids";
 import ReviewExternal from "./ReviewExternal";
 import ReviewInfomedia from "./ReviewInfomedia";
 import ReviewLibrarian from "./ReviewLibrarian";
+import ReviewSkeleton from "./ReviewSkeleton";
 
 export interface MaterialReviewsProps {
-  listOfReviews: Array<LibrariansReview | ExternalReview | InfomediaReview>;
+  pids: Pid[];
+  dataCy?: string;
 }
 
 export const MaterialReviews: React.FC<MaterialReviewsProps> = ({
-  listOfReviews
+  pids,
+  dataCy = "material-reviews"
 }) => {
+  const { data, isLoading } = useGetReviewManifestationsQuery({
+    pid: pids
+  });
+
+  if (isLoading) {
+    return <ReviewSkeleton />;
+  }
+
+  if (!data) {
+    return null;
+  }
```

Comment:
Have you tried throttling the network? Returning a null here could potentially lead to a glitch in the rendering.

Reply:
Yes I have tried Slow 3G :) all works fine. 
It's because if it's still loading we just show the skeleton.